tests/console/yast2_i.pm

```diff
@@ -98,8 +98,7 @@ sub run() {
     clear_console;                           # clear screen to see that second update does not do any more
     assert_script_run("rpm -e $pkgname");    # erase $pkgname
     script_run("echo mark yast test", 0);    # avoid zpper needle
-    script_run("rpm -q $pkgname",     0);
-    assert_screen("yast-package-$pkgname-not-installed", 1);
+    assert_script_run("rpm -q $pkgname");
```

Comment:
I forgot to mark the PR as "[WIP]" as my test run was not complete. Nevertheless what I tested in before is just call `rpm -q foo` which gives an exit status of 1 as expected and `rpm -q bash` gives an exit status of 0 plus a corresponding output.